lmod/lmod.py

```diff
@@ -361,4 +360,11 @@ class LmodTestJavaMemory(LmodTestBase):
     def assert_output(self):
         return sn.assert_found(r'^True$', self.stdout)
 
+@rfm.simple_test
+class LmodTestCachedLoads(LmodTestBase):
```

Comment:
shouldn't we best have two: one with explicit cached load on and one where it's off?